wdl/TrainGqRecalibrator.wdl

```diff
@@ -0,0 +1,234 @@
+                fix_vcf_args = fix_vcf_args,
+                sv_utils_docker=sv_utils_docker,
+                index_output_vcf=true
+        }
+    }
+
+    if(!defined(train_vcf_index) && !fix_vcf) {
+        call IndexVcf {
+            input:
+                vcf=train_vcf,
+                sv_base_docker=sv_base_docker
+        }
+    }
+
+    File train_vcf_ = select_first([FixVcf.fixed_vcf, train_vcf])
+    File train_vcf_index_ = select_first([FixVcf.fixed_vcf_index, train_vcf_index, IndexVcf.index_file])
+
+    call GetTruthOverlap.GetTruthOverlap as GetTruthOverlap {
+        input:
+            test_vcfs=[train_vcf_],
+            test_vcf_indices=[train_vcf_index_],
+            truth_vcfs=truth_vcfs,
+            truth_vcf_indices=truth_vcf_indices,
+            vapor_files=vapor_files,
+            ped_files=[ped_file],
+            optimal_overlap_cutoffs=optimal_overlap_cutoffs,
+            get_truth_overlap_args=get_truth_overlap_args,
+            sv_utils_docker=sv_utils_docker,
+            sv_base_docker=sv_base_docker
+    }
+
+    call GetTruthOverlap.GetVcfSize as GetVcfSize {
+        input:
+            vcf=train_vcf_,
+            vcf_index=train_vcf_index_,
+            sv_base_docker=sv_base_docker
+    }
+
+    call TrainGqRecalibratorTask {
+        input:
+            train_vcf=train_vcf_,
+            train_vcf_index=train_vcf_index_,
+            ped_file=ped_file,
+            truth_file=GetTruthOverlap.truth_overlap_info,
+            genome_tracts=genome_tracts,
+            gq_recalibrator_model_file=gq_recalibrator_model_file,
+            train_args=train_args,
+            gatk_docker=gatk_docker,
+            num_entries=GetVcfSize.num_entries
+    }
+
+    output {
+        File clean_vcf=train_vcf_
+        File clean_vcf_index=train_vcf_index_
+        File truth_overlap_info = GetTruthOverlap.truth_overlap_info
+        File output_optimal_overlap_cutoffs = GetTruthOverlap.output_optimal_overlap_cutoffs
+        File output_gq_recalibrator_model_file = TrainGqRecalibratorTask.output_gq_recalibrator_model_file
+    }
+}
+
+
+task FixVcf {
+    input {
+        File vcf
+        Array[String] fix_vcf_args
+        String sv_utils_docker
+        Boolean index_output_vcf = true
+    }
+
+    String fixed_vcf_name = sub(sub(basename(vcf), ".gz$", ""), ".vcf$", "_fixed.vcf.gz")
+    String index_file_name = fixed_vcf_name + ".tbi"
+    String args_str = if length(fix_vcf_args) > 0 then sep(" ", fix_vcf_args) else ""
+
+    Float uncompress_scale = 10
+    Int disk_gb = 1000 + round((2 + uncompress_scale) * size(vcf, "GiB"))
+    Int mem_gb_overhead = 2
+    Float mem_scale = 2.0
+    Float mem_gb = mem_gb_overhead + mem_scale * size(vcf, "GiB")
+
+    runtime {
+        docker: sv_utils_docker
+        cpu: 1
+        preemptible: 3
+        max_retries: 0
+        memory: mem_gb + " GiB"
+        disks: "local-disk " + disk_gb + " HDD"
+    }
+
+    command <<<
+        set -eu -o pipefail
```

Comment:
I don't feel strongly about this, so I just went ahead and changed it to the house style you suggested. I just want to say, I tend to like the original form because it wasn't obvious to me (before I did a ton of bash) that the "o" and the pipefail go together. So for instance, `set -euox pipefail` doesn't work and neither does `set -eu pipefail`. I like keeping them separate to make that more clear. Anyway, if we change some, we should change all. So I set these to house style.